ci-operator/config/jianzhangbjz/learn-operator/jianzhangbjz-learn-operator-master.yaml

```diff
@@ -0,0 +1,84 @@
+base_images:
+  base:
+    name: "4.7"
+    namespace: ocp
+    tag: base
+  cli:
+    name: "4.7"
+    namespace: ocp
+    tag: cli
+  operator-registry:
+    name: "4.7"
+    namespace: ocp
+    tag: operator-registry
```

Comment:
This does not seem to be used anywere?

Reply:
Yes, thanks! I guess no need it anymore, removed it.